hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala

```diff
@@ -157,6 +159,9 @@ class ServiceBackend(
       if (backendContext.workerMemory != "None") {
         resources = resources.merge(JObject(("memory" -> JString(backendContext.workerMemory))))
       }
+      if (backendContext.storageRequirement != "0Gi") {
```

Comment:
Yes, this number is set in `service_backend.py`. There is also no need for any compatibility requirements between `service_backend.py` and `ServiceBackend.scala` so this can change as we wish as long as they agree with each other in the same commit.